test/index.js

```diff
@@ -474,4 +473,107 @@ describe('HauteCouture', () => {
         });
     });
 
+    it('allows options to be used as a callback', (done) => {
+
+        const opCbSrv = new Hapi.Server();
```

Comment:
Any reason the server and plugin variables in each test are named specifically for the test?